thirdparty/faiss/faiss/IndexIVF.h

```diff
@@ -71,6 +71,7 @@ struct Level1Quantizer {
 struct SearchParametersIVF : SearchParameters {
     size_t nprobe = 1;    ///< number of probes at query time
     size_t max_codes = 0; ///< max nb of codes to visit to do a query
+    bool ensure_topk_full = false; ///< indicate whether we make sure topk result is full
```

Comment:
I'd add more comments here. It did not become clear what this parameter does before I read the code :)

Reply:
Sure, I will add more comments in the next pr.